### PR TITLE
Create tropical-animal-health-and-production.csl

### PR DESCRIPTION
**Pull request by damnation333**
Code was here, but had to be modified: https://forums.zotero.org/discussion/42877/style-request-tropical-animal-health-and-production/
I am struggling with the chapters. How can I get it to display the page number at the very end? When I add it into the "publisher" macro it duplicates it for normal article-journals, too. 
The rest all works.

### CODE REVIEW

File: tropical-animal-health-and-production.csl [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="chapter-info">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <group delimiter=", ">
+          <group delimiter=" ">
```
Comment (rmzelle) [inline tropical-animal-health-and-production.csl]: The outer group doesn't do anything here.

**Comment by rmzelle** [comment]
> I am struggling with the chapters. How can I get it to display the page number at the very end? When I add it into the "publisher" macro it duplicates it for normal article-journals, too.

Well, there are a bunch of ways. You could make the output in the macro conditional on item type (e.g. `<if type="chapter">`), or you could split up the `<layout/>` section in the bibliography by item type (again with a conditional).

**Comment by damnation333** [comment]
That works now, but I guess it doens't validate now because there is no <else> following the <if>. How can I make it have that <else> without doing anything?

**Comment by rmzelle** [comment]
The problem is that `<if/>` needs to be wrapped inside a `<choose/>`. You can't use it by itself.

**Pull request merged**